### PR TITLE
Add a useful installed program for practical use

### PR DESCRIPTION
**Pull request by tbrowder**
If this doesn't suit you, I will produce a new module. Something named like 'App::GitStat'.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Actions Status](https://github.com/lizmat/Git-Status/workflows/test/badge.svg)](https://github.com/lizmat/Git-Status/actions)
+[![Actions Status](https://github.com/tbrowder/Git-Status/workflows/test/badge.svg)](https://github.com/tbrowder/Git-Status/actions)
```
Comment (lizmat) [inline README.md]: This change feels incorrect?

Comment (tbrowder) [reply README.md]: Okay, see my edit just now. Oh, yes, that should be lizmat for sure.

But, if accepted, your next build would change it back.

I just pushed a change...but I don't see it here yet...

**Comment by tbrowder** [comment]
The change is there now.

**Comment by lizmat** [comment]
I've been thinking about this, but I think I want to keep `Git::Status` simple, and not `App`y at all.

I think you can take most of this code verbatim and put it into a separate module with `Git::Status` as a dependency.

Thank you for your work!  Looking forward to a new module in the ecosystem!

**Comment by tbrowder** [comment]
I understand, Liz, I didn't want to go on my own unless you agreed. See ya! -Tom

**Pull request closed without merging**